proto/wg/cosmo/platform/v1/platform.proto

```diff
@@ -2813,4 +2837,10 @@ service PlatformService {
   rpc CreateBillingPortalSession(CreateBillingPortalSessionRequest) returns (CreateBillingPortalSessionResponse) { }
   // Upgrade the current plan
   rpc UpgradePlan(UpgradePlanRequest) returns (UpgradePlanResponse) { }
+
+  // Router Compatibility Versions
+  rpc ListRouterCompatibilityVersions(ListRouterCompatibilityVersionsRequest) returns (ListRouterCompatibilityVersionsResponse) { }
+
+  // Graph
+  rpc SetGraphRouterCompatibilityVersion(SetGraphRouterCompatibilityVersionRequest) returns (SetGraphRouterCompatibilityVersionResponse) { }
```

Comment:
`..Version` or `...Versions`. it should be consistent with request and response types

Reply:
I don't understand.

Reply:
@StarpTech it's list VersionS (plural) because there is more than one. Get and Set is Version (singular)